msm/src/circuit_design/composition.rs

```diff
@@ -70,6 +73,29 @@ pub trait MPrism {
     fn re_get(&self, target: Self::Target) -> Self::Source;
 }
 
+/// Identity `MPrism` from any type `T` to itself.
+#[derive(Clone, Copy, Debug)]
+pub struct IdMPrism<T>(pub PhantomData<T>);
```

Comment:
Do I understand correctly that it will be used to represent compositions of circuits that "have the same shape"?